EK-TM4C123GXL/keypad/keypad.h

```diff
@@ -1,6 +1,24 @@
 #ifndef KEYPAD_H_
 #define KEYPAD_H_
 
+// Truth Table
+#define Zero 0x21
```

Comment:
Capitalize defines..
#define ZERO 0x21
And consider better names... ie. #define KP_ZERO 0x21
Where KP means keypad.